ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -13,59 +13,97 @@ Test Teardown     Resources Test Teardown
 
 *** Variables ***
 @{LIST_OF_IDS_FOR_COMBINATIONS}         documentation--check-box    Red Hat managed--check-box
+${SUCCESS_STEP}              h3[normalize-space(@class="pf-c-title pf-m-md pfext-quick-start-task-header__title
+...                           pfext-quick-start-task-header__title-success")]
+
 
 
 *** Test Cases ***
-Verify Quick Starts Work As Expected
+Verify Quick Starts Work As Expected on Yes and Restart
+    [Documentation]   Verify the Quickstarts are completed successfully
+    ...    when all steps are marked as yes and restarted later
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1306    ODS-1308
+    ...     ODS-1406    ODS-1405
+    Set Quick Starts Elements List Based on RHODS Type
+    FOR    ${qsElement}    IN    @{quickStartElements}
+        Verify Quick Starts Work As Expected When All Steps Are Marked As Yes   ${qsElement}
+        Verify Quick Starts Work As Expected When Restarted And Left In Between    ${qsElement}
+        Verify Quick Starts Work As Expected When Restarting The Previous One    ${qsElement}
+    END
+
+Verify Quick Starts Work When All Steps Are Skipped
+    [Documentation]   Verify the Quickstarts work fine when all
+    ...    steps are skipped
     [Tags]  Sanity
     ...     Tier1
-    ...     ODS-1166    ODS-1306
-    ...     ODS-1307    ODS-1308
-    ...     ODS-1402    ODS-1403
-    Verify Quick Starts Work As Expected When All Steps Are Marked As Yes   create-jupyter-notebook-anaconda
-    Verify Quick Starts Work As Expected When Restarting The Previous One   create-jupyter-notebook-anaconda
-    Verify Quick Starts Work As Expected When All Steps Are Skipped         create-jupyter-notebook
-    Verify Quick Starts Work As Expected When At Least One Step Is Skipped      deploy-python-model
-    Verify Quick Starts Work As Expected When One Step Is Marked As No  openvino-inference-notebook
-
-Verify External Links In Quick Starts Are Not Broken
+    ...     ODS-1402
+    Set Quick Starts Elements List Based on RHODS Type
+    FOR    ${qsElement}    IN    @{quickStartElements}
+        Verify Quick Starts Work As Expected When All Steps Are Skipped    ${qsElement}
+    END
```

Comment:
we try avoiding branching/looping in the test case itself, like per [Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.sxbtbpwa2iba)

Reply:
Moved this looping logic to the Keywords